### DISCUSSION TITLE
How can I add note to specific folder🙏🏻

### DISCUSSION THREAD

**Discussion by SirapraphatS** [coddingtonbear/obsidian-web#166] [Q&A] [answered]
It good!!, but I want to put my note into specific folder

**Comment by coddingtonbear** [comment, marked as answer]
All you'll need to do is just need to adjust the "API URL" field in a specific spot in your template.  Here's an example of a template I'm using for an ongoing apartment hunt as part of a move I have coming up:

![image](https://github.com/coddingtonbear/obsidian-web/assets/527661/108e4288-43c1-4e19-9a3a-c431ebdfcb63)

If you look at the "API URL" field, you'll see that there's a value entered there that starts with `/vault/`.  The bit after `/vault/` is how you tell the underlying API where you would like the file to be placed in your *vault*.

I think the above is everything you're looking for, but if you were very curious, there are lots of other things you can do in that "API URL" field.  You can see what other options are available by having a look at the [Obsidian Local REST API Docs](https://coddingtonbear.github.io/obsidian-local-rest-api/) .  If that list looks overwhelming, in your use case you're probably looking for entries there marked `PUT`, and I can't imagine *any* usecases of Obsidian Web that might use anything outside of `PUT`, `POST`, or `PATCH`.

Good luck!
